src/main/resources/db/migration/V13__system_melding.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE SYSTEM_MELDING (
```

Comment:
Kan slås sammen med V14, siden jeg merget inn en V13 for beslutterfunksjon nå nylig